app/components/articleSearch/components/pagination/pagination.scss

```diff
@@ -0,0 +1,74 @@
+@import "../../../../variables";
+
+.pagination {
+  display: flex;
+  justify-content: center;
+  margin-top: 69px;
+  margin-bottom: 79px;
+
+  .pageItem {
+    margin-right: 10px;
+    width: 44px;
+    height: 44px;
+    border: solid 1px $space_gray16;
+    font-size: 16px;
+    color: $space_gray3;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    transition: border-color 250ms cubic-bezier(0.4, 0, 0.2, 1), color 250ms cubic-bezier(0.4, 0, 0.2, 1);
```

Comment:
Following by Material design guide, desktop animation should be <= 200ms for avoiding awful UX.

Reply:
I don't think what Google said is the perfect answer but I believe we should talk about this with our designer.

Reply:
I agree